pkg/tsdb/cloudwatch/routes/regions.go

```diff
@@ -0,0 +1,45 @@
+package routes
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/models"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/services"
+)
+
+const (
+	defaultRegion = "default"
+)
+
+func RegionsHandler(ctx context.Context, pluginCtx backend.PluginContext, reqCtxFactory models.RequestContextFactoryFunc, parameters url.Values) ([]byte, *models.HttpError) {
+	service, err := newRegionsService(ctx, pluginCtx, reqCtxFactory, defaultRegion)
+
+	if err != nil {
+		return nil, models.NewHttpError("Unexpected error while fetching regions", http.StatusInternalServerError, err)
+	}
+
+	regions, err := service.GetRegions()
+	if err != nil {
+		return nil, models.NewHttpError("Unexpected error while fetching regions", http.StatusInternalServerError, err)
+	}
+
+	regionsResponse, err := json.Marshal(regions)
+	if err != nil {
+		return nil, models.NewHttpError("Unexpected error while fetching regions", http.StatusInternalServerError, err)
+	}
```

Comment:
Probably make these error strings different so that log debugging can identify just one path